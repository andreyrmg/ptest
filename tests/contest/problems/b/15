100 10
8 92 43 99 6 90 16 81 51 7
2 71 48
2 53 97
2 65 46
2 92 84
1 7
1 89
2 5 26
2 69 59
2 63 75
2 43 58
3 19 81 83
3 21 32 69
2 100 74
3 93 76 89
2 60 36
2 39 37
1 20
2 57 100
2 11 75
2 17 36
4 12 58 35 82
3 39 29 44
1 83
2 63 45
3 56 54 65
2 7 50
1 38
1 64
1 22
1 90
3 78 33 52
3 12 78 79
1 31
1 95
2 21 77
2 20 15
1 16
2 27 98
4 16 22 90 50
2 74 66
2 67 49
2 52 68
3 10 80 96
2 22 85
1 24
1 3
2 88 60
1 1
1 41
3 26 39 80
2 64 53
3 42 31 73
3 2 51 86
2 25 88
1 64
3 25 97 67
2 18 62
3 21 10 72
1 8
2 15 47
1 64
2 57 75
3 9 24 69
6 28 51 55 61 74 91
2 3 25
4 40 70 71 90
2 41 56
1 42
3 8 63 12
1 66
5 1 66 92 98 99
1 58
2 52 87
3 40 64 13
3 19 62 9
1 14
1 35
2 31 32
1 32
2 43 50
1 11
1 21
3 23 11 94
1 4
1 44
2 53 95
1 73
2 47 54
3 6 14 93
3 30 39 66
1 64
2 4 71
2 14 89
1 83
2 34 86
1 43
2 56 2
2 38 71
1 71
2 13 18
