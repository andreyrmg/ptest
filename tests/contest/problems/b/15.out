22 9 7 1 5 10 2 9 32 3 